fastlane_bot/bot.py

```diff
@@ -803,15 +808,16 @@ def calculate_profit(
             self.ConfigObj.logger.debug(f"[bot.calculate_profit sorted_price_curves] {sorted_price_curves}")
             if len(sorted_price_curves)>0:
                 fltkn_eth_conversion_rate = sorted_price_curves[0][-1]
-                best_profit_eth = Decimal(str(best_profit_fl_token)) / Decimal(str(fltkn_eth_conversion_rate))
+                flashloan_fee_amt_eth = Decimal(str(flashloan_fee_amt_fl_token)) / Decimal(str(fltkn_eth_conversion_rate))
+                best_profit_eth = Decimal(str(best_profit_fl_token)) / Decimal(str(fltkn_eth_conversion_rate)) - flashloan_fee_amt_eth
                 self.ConfigObj.logger.debug(f"[bot.calculate_profit] {fl_token, best_profit_fl_token, fltkn_eth_conversion_rate, best_profit_eth, 'ETH'}")
             else:
                 self.ConfigObj.logger.error(
                     f"[bot.calculate_profit] Failed to get conversion rate for {fl_token} and {self.ConfigObj.WRAPPED_GAS_TOKEN_ADDRESS}. Raise"
                 )
                 raise
         else:
-            best_profit_eth = best_profit_fl_token
+            best_profit_eth = best_profit_fl_token - flashloan_fee_amt_fl_token
```

Comment:
Oh, I see that my suggestion above is actually applied here, so the question arises - why are you **currently** calculating `best_profit_eth` in two different ways (dividing by the ETH conversion rate in one place, and not diving by the ETH conversion rate in another place)?

Reply:
This is why it helps to look at more than just the diffs. If the flashloan is not in ETH/WETH, then it is converted into ETH with:
```python
flashloan_fee_amt_eth = Decimal(str(flashloan_fee_amt_fl_token)) / Decimal(str(fltkn_eth_conversion_rate))
```

otherwise, if the flashloan is in ETH/WETH, then:
```python
best_profit_eth = best_profit_fl_token - flashloan_fee_amt_fl_token
```

Because of this, I'm not really sure where _the other place_ you suggest above would be? (ie the value is used here.